paths.json

```diff
@@ -1,6 +1,9 @@
 {
   "mappings": [
-    "/content/pricefx/en:/"
+    "/content/pricefx/:/",
+    "/content/pricefx/en:/",
+    "/content/exlm/en/configuration/redirects.json",
```

Comment:
@mayur-satav Can you take out the "/content/exlm/en/configuration/redirects.json", ?

Reply:
done